ci/testsuite-result.json

```diff
@@ -20022,6 +20022,127 @@
     ],
     "time": null
   },
+  {
+    "command": "host info foo",
+    "command_filter": null,
+    "command_filter_negate": false,
+    "command_issued": "host info foo # should work and show both IPs",
+    "ok": [],
+    "warning": [],
+    "error": [],
+    "output": [
+      "Name:         foo.example.org",
+      "Contact:      ",
+      "A_Records     IP            MAC",
+      "              10.0.0.5      aa:bb:cc:dd:ee:ff",
+      "AAAA_Records  IP            MAC",
+      "              2001:db8::5   <not set>",
```

Comment:
Need to fix MAC association here

Reply:
Do we? I mean, it's implied due to the way things work at UiO, but the association is not explicitly set and I don't think that was the intent.

Reply:
Ok, let's leave it as it is.